fusor-ember-cli/app/templates/components/cancel-back-next.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="col-md-9 rhci-steps-buttons">
+<div class="col-xs-8 col-sm-8 col-md-9 col-lg-10 col-xs-push-4 col-sm-push-4 col-md-push-3 col-lg-push-2 sidebar-pf-right rhci-steps-buttons cancel-back-next">
```

Comment:
@cfchase, I just used the small bootstrap grid logic as in [components/wizard-step.hbs](https://github.com/fusor/fusor/blob/master/fusor-ember-cli/app/templates/components/wizard-step.hbs)